LocalParks.API/Startup.cs

```diff
@@ -0,0 +1,121 @@
+using LocalParks.Data;
+using LocalParks.Infrastructure;
+using LocalParks.Infrastructure.Models;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using System;
+using System.Collections.Generic;
+
+namespace LocalParks.API
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddLocalParksData(Configuration.GetConnectionString("LocalParks"))
+                 .AddLocalParksInfrastructure(Configuration.GetSection("Tokens"));
+
+            services.AddAutoMapper(typeof(ParkProfile));
+
+            services.AddCors(options =>
+            {
+                options.AddPolicy("Open", builder => builder.AllowAnyOrigin().AllowAnyHeader().AllowAnyMethod());
```

Comment:
Be more restrictive here as cor's is designed to stop someone using your api's you can do this in azure instead also up to you.

Reply:
Yeah I'll change it to only the Client App endpoint, thanks!